default.nix

```diff
@@ -102,112 +99,77 @@ rec {
     meta = { };
     buildInputs = deps;
     buildPhase = ''
-      bash typeCheck.sh
+      agda --profile=modules Everything.agda \
+      | tee typecheck.log \
+      | sed '/Checking/!d'
```

Comment:
I put the piping through `tee` and `sed` so we have some feedback while agda is typechecking but not the time it took per module at the end.

Reply:
nice